docs/src/systems/boundary.md

```diff
@@ -93,7 +97,20 @@ where the sum is over all fluid particles, ``\rho_f`` and ``p_f`` denote the den
     AdamiPressureExtrapolation
 ```
 
-#### 4. [`PressureZeroing`](@ref)
+#### 2. [`BernoulliPressureExtrapolation`](@ref)
+Identical to [`AdamiPressureExtrapolation`](@ref), but it adds the dynamic pressure component of the Bernoulli equation:
+```math
+\frac{1}{2} \, \rho_{\text{neighbor}} \left( \frac{ (\mathbf{v}_f - \mathbf{v}_{\text{neighbor}}) \cdot (\mathbf{x}_f - \mathbf{x}_{\text{neighbor}}) }{ \left\| \mathbf{x}_f - \mathbf{x}_{\text{neighbor}} \right\| } \right)^2 \times \text{factor}
```

Comment:
1. This now doesn't include the sign of the velocity difference. This seems wrong. The pressure should only increase when fluid and solid are moving towards each other, not away from each other, no?
2. I don't find it very clear where this term is added exactly. Is it just added to the pressure of the Adami method? Is this even a pressure physically?

Reply:
1. what are you talking about? That is the equation:
![ql_e67553e12348838d187ae5fc26861c0b_l3](https://github.com/user-attachments/assets/0b0c96ea-14df-4a47-912c-a2bc9f31b3c4)

2. It has the units of a pressure. Is this a physical pressure no.